apps/dashr-wind-steaming/app.R

```diff
@@ -0,0 +1,443 @@
+
+library(dashR)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+library(purrr)
+library(dplyr)
+library(DBI)
+library(RSQLite)
+library(glue)
+library(data.table)
+library(plotly)
+library(readr)
+library(VGAM)
+
+
+appName <- Sys.getenv("DASH_APP_NAME")
+pathPrefix <- sprintf("/%s/", appName)
+
+Sys.setenv(DASH_ROUTES_PATHNAME_PREFIX = pathPrefix,
+           DASH_REQUESTS_PATHNAME_PREFIX = pathPrefix)
+
+setwd("/app/apps/dashr-opiod-endemic")
+setwd("/app/apps/dashr-wind-streaming")
+
+DB_FILE = "/app/apps/dashr-wind-streaming/db/wind-data.db" 
+
+get_wind_data <- function(start, end) {
+  # Query wind data rows between two ranges
+  # :params start: start row id
+  # :params end: end row id
+  # :returns: pandas dataframe object
+  con <- dbConnect(SQLite(), DB_FILE)
+  res <- dbSendQuery(con, glue("SELECT Speed, SpeedError, Direction FROM Wind WHERE rowid > {start} AND rowid <= {end}"))
+  df <- dbFetch(res)
+  return(df)
+}
+
+get_wind_data_by_id <- function(id){
+  # Query a row from the Wind Table
+  # :params id: a row id
+  # :returns: pandas dataframe object
+  con <- dbConnect(SQLite(), DB_FILE)
+  res <- dbSendQuery(con, glue("SELECT * FROM Wind WHERE rowid = {id}"))
+  df <- dbFetch(res)
+  return(df)
+}
+
+get_current_time <- function(){
+  # Helper function to get the current time in seconds
+  total_time <- as.integer(as.numeric(Sys.time()) / 216000)
```

Comment:
Whenever there's a value in your code like this, can be helpful to spell it out. 

In other words, if 216000 is a product, do the computation inline so that it's a little more obvious where the number comes from 🙂